sdk/python/feast/repo_operations.py

```diff
@@ -118,6 +118,13 @@ def apply_total(repo_config: RepoConfig, repo_path: Path):
     registry._initialize_registry()
     sys.dont_write_bytecode = True
     repo = parse_repo(repo_path)
+    repo = ParsedRepo(
+        feature_tables=repo.feature_tables,
+        entities=infer_entity_value_type_from_feature_views(
```

Comment:
I think we are missing inferencing when using the Python SDK https://github.com/feast-dev/feast/blob/master/sdk/python/feast/feature_store.py#L227